app/components/provider_interface/completed_offer_summary_component.rb

```diff
@@ -12,14 +12,14 @@ def rows
           value: course_option.course.name_and_code,
           action: 'Change',
           change_path: change_course_path },
-        { key: 'Full time or part time',
-          value: course_option.study_mode.humanize,
-          action: 'Change',
-          change_path: change_study_mode_path },
         { key: 'Location',
           value: course_option.site.name_and_address,
           action: 'Change',
           change_path: change_location_path },
+        { key: 'Full time or part time',
```

Comment:
I'm a little confused, the card says 'Full time/part time should come before locations in all summary displays'. Is this change required?

Reply:
I think in my attempt to get through all the make offer components I missed that some had been implemented with the details in the correct order :)

Thanks for flagging. Sorted now.